nailgun/entities.py

```diff
@@ -2457,6 +2457,8 @@ class OperatingSystem(
     * architecture
```

Comment:
1) `architecture`, `medium` and `ptable` are valid fields and are listed in the API doc for all three main operations - `GET`, `POST` and `PUT`
2) not sure that we need to mention that long URL `<https://bugzilla.redhat.com/show_bug.cgi?id=1290359>` as it was not mention 5 lines before
3) we should also mention `description` in some way